lib/components/admin/styled.js

```diff
@@ -145,7 +145,7 @@ export const Text = styled.span`
 
 export const Val = styled.span`
   :empty:before {
-    color: grey;
+    color: #685C5C; /* from "gray" to meet WCAG */
```

Comment:
We probably don't need a comment about the previous value here.

Reply:
Removed in 6188d3ac981359220491cc83995fe7f351d9efb2